.github/workflows/storybook release.yml

```diff
@@ -0,0 +1,25 @@
+name: Storybook Release Workflow
```

Comment:
Maybe we should replace the previous `storybook` workflow instead of creating a new one? The storybook should reflect the state of published package only I think.

(This comment refers to filename)

Reply:
All your comments will be discussed and resolved in the meet, I left both workflows and release packages to make comparison easier